projects/core/src/lib/filter/helpers/cql-filter.helper.spec.ts

```diff
@@ -0,0 +1,145 @@
+import { FilterGroupModel } from '../models/filter-group.model';
+import { FeatureAttributeTypeEnum } from '@tailormap-viewer/api';
+import { FilterConditionEnum } from '../models/filter-condition.enum';
+import { CqlFilterHelper } from './cql-filter.helper';
+
+describe('CQLFilterHelper', () => {
+
+  test('should create a basic CQL filter', () => {
+    const filterGroup: FilterGroupModel = {
+      id: '1',
+      layerId: 1,
+      filters: [{
+        id: '1',
+        caseSensitive: false,
+        invertCondition: false,
+        attribute: 'attribute',
+        attributeType: FeatureAttributeTypeEnum.STRING,
+        condition: FilterConditionEnum.STRING_LIKE_KEY,
+        value: ['value'],
+      }],
+      operator: 'AND',
+      source: 'SOME_COMPONENT',
+    };
+    const filters = CqlFilterHelper.getFilters([filterGroup]);
+    expect(filters.get(1)).toBe('(attribute LIKE \'%value%\')');
+  });
+
+  test('combine multiple filters into a CQL filter', () => {
+    const filterGroup: FilterGroupModel = {
+      id: '1',
+      layerId: 1,
+      filters: [{
+        id: '1',
+        caseSensitive: false,
```

Comment:
actually LIKE is case sensitive, ILIKE is not, see https://docs.geoserver.org/stable/en/user/filter/ecql_reference.html#predicate

```suggestion
        caseSensitive: true,
```